### PR TITLE
Day2 SNO cluster worker nodes

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/24569

### CODE REVIEW

**Comment by jelkosz** [comment]
lgtm

File: multicluster_engine/cluster_lifecycle/create_cluster_on_prem.adoc [hunk 1]
```diff
@@ -51,7 +51,10 @@ The *OpenShift version* identifies the version of the {ocp-short} image that is
 
 When you select an OpenShift version that is 4.9 or later, an option to select *Install single node OpenShift (SNO)* is displayed. A single-node OpenShift cluster contains a single node which hosts the control plane services and the user workloads. You cannot add additional nodes to a single-node OpenShift cluster after it is created. 
 
-If you want your cluster to be a single-node OpenShift cluster, select the single-node OpenShift option. 
+If you want your cluster to be a single-node OpenShift cluster, select the single-node OpenShift option. You can add additional workers to single-node OpenShift clusters by completing the following steps:
```
Comment (dockerymick) [inline multicluster_engine/cluster_lifecycle/create_cluster_on_prem.adoc]: Change: If you want your cluster to be a single-node OpenShift cluster, select the single-node OpenShift option.

TO

If you want your cluster to be a single-node OpenShift cluster, select the **single-node OpenShift** option.

File: multicluster_engine/cluster_lifecycle/create_cluster_on_prem.adoc [hunk 2]
```diff
@@ -51,7 +51,10 @@ The *OpenShift version* identifies the version of the {ocp-short} image that is
...
+
+. From the console, navigate to *Infrastructure* > *Clusters* and select the name of the cluster that you created or want to access. 
+. Select *Actions* > *Add hosts* to add additional workers.
```
Comment (swopebe) [inline multicluster_engine/cluster_lifecycle/create_cluster_on_prem.adoc:58]: beautiful

File: release_notes/whats_new.adoc [hunk 3]
```diff
@@ -41,6 +41,8 @@ Learn more in the link:../console/console.adoc#console-overview[Console overview
 
 * The navigation menu option to access the infrastructure environments changed from *Infrastructure environment* to *Host inventory*.
 
+* You can add additional workers to single-node OpenShift clusters created using the Central Infrastructure Management service. See link:../multicluster_engine/cluster_lifecycle/create_cluster_on_prem.adoc#on-prem-creating-your-cluster-with-the-console[Creating your cluster with the console] for more details.
+
```
Comment (swopebe) [inline release_notes/whats_new.adoc:45]: By using, or that is using....it's important to use the one you mean here.

**Review by swopebe — CHANGES_REQUESTED** [review]
See comments.

File: release_notes/whats_new.adoc (hunk 3 above)
Comment (oafischer) [reply release_notes/whats_new.adoc:45]: Changed to with, I think that should still be correct from a technical standpoint. If not, I think "by using" would be the closest.

**Review by swopebe — APPROVED** [review]

**Pull request merged**